src/tests/simple_common.h

```diff
@@ -0,0 +1,101 @@
+/**
+ * (C) Copyright 2016-2021 Intel Corporation.
```

Comment:
Is this a new file?  Should the copyright only be 2021?

Reply:
yes, will fix